### PR TITLE
Devise login

### PR DESCRIPTION
**Pull request by alappies**

### CODE REVIEW

File: app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -1,4 +1,5 @@
-<h2>Log in</h2>
+<div id="container">
+<h2 id="log-in">Log in</h2>
```
Comment (zan-rosenthal) [inline app/views/devise/sessions/new.html.erb]: This should be indented 2 space.

File: app/views/devise/sessions/new.html.erb [hunk 2]
```diff
@@ -24,3 +25,4 @@
 <% end %>
 
 <%= render "devise/shared/links" %>
+</div>
```
Comment (zan-rosenthal) [inline app/views/devise/sessions/new.html.erb]: Need a newline at the end of this file. Then you can merge.

File: app/views/devise/sessions/new.html.erb (hunk 1 above)
Comment (zan-rosenthal) [reply app/views/devise/sessions/new.html.erb]: Also, actually, looks like everything inside of the div isn't indented. Any time an element is nested in another element, it should be indented 2 lines form the parent element's indentation.

**Review by zan-rosenthal — CHANGES_REQUESTED** [review]

**Review by zan-rosenthal — APPROVED** [review]

File: app/views/devise/sessions/new.html.erb (hunk 2 above)
Comment (alappies) [reply app/views/devise/sessions/new.html.erb]: Taken care of thank you for the quick review!!

**Pull request merged**